week3/index.js

```diff
@@ -12,108 +12,108 @@ app.listen(PORT, () => {
     console.log(`Server listening on port ${PORT}`);
 });
 
+app.use(bodyParser.json());
 
 
-function handleRequest() {
+app.get("/todo-api/list", (req, res) => {
 
-    app.use(bodyParser.json());
-
-
-    app.get("/list", (req, res) => {
-        readTodosFile().then(data => {
-            if (!data || data === "[]") res.send("There are no items on the list");
-  
-            res.send(data);
-
-        }).catch(err => console.error(err));   
+    readTodosFile().then(data => {
+        if (!data || data === []) {
+            res.json({ message: "There are no items on the list" });
```

Comment:
You probably want to `return` here, or put the next line (`res.json(data)`) in the else block, so that they don't both get triggered when the list is empty.